action.yml

```diff
@@ -47,6 +47,14 @@ inputs:
     description: 'Action will update pull request status. Default: true'
     required: false
     default: 'true'
+  copr:
+    description: 'Name of copr to use for the artifacts'
+    required: false
+    default: 'epel-7-x86_64'
+  env_artifacts:
+    description: 'Environment artifacts for test env, separated by ;'
```

Comment:
As long as only `fedora-copr-build` artifacts are supported, maybe it should be mentioned in the input description?